### PR TITLE
[PE-4] Landing page text update for Project Singular

### PR DESCRIPTION
**Pull request by kyuksel**
The PR changes
- involve no functional changes;
- add the text highlighted on the screenshot below;
- can be enabled via `isProjectSingular: true` flag while running locally.

image

**EDIT:** I had somehow missed half of the requested changes :( The latest looks like the following:

image

### CODE REVIEW

**Review by cahrens — APPROVED** [review]

**Review by nawatts — APPROVED** [review]

**Comment by kyuksel** [comment]
@cahrens @nawatts Thank you for the review! I'm afraid I'll have to request re-reviews as I'd somehow missed half of the requested changes :( It initially looked too straightforward so I didn't open the PR as draft until getting the ✅ from the PO. So sorry about that.

File: src/components/common.js [hunk 1]
```diff
@@ -54,39 +54,40 @@ const styles = {
   }
 }
```
Comment (kyuksel) [inline src/components/common.js:56]: --- Beginning of unrelated reformatter changes ---

File: src/components/common.js [hunk 2]
```diff
@@ -423,8 +424,7 @@ export const UnlinkFenceAccount = ({ linkText, provider }) => {
             message: `Successfully unlinked your account from ${provider.name}`,
             timeout: 30000
           })
-        }
-        )
+        })
```
Comment (kyuksel) [inline src/components/common.js:427]: --- End of unrelated reformatter changes ---

File: src/components/common.js [hunk 3]
```diff
@@ -510,6 +510,11 @@ export const Switch = forwardRefWithName('Switch', ({ onChange, onLabel = 'True'
 export const HeroWrapper = ({ showMenu = true, bigSubhead = false, children }) => {
   const heavyWrapper = text => bigSubhead ? b({ style: { whiteSpace: 'nowrap' } }, [text]) : text
 
+  const endText = Utils.cond(
+    [isProjectSingular(), () => div(['and ', heavyWrapper('collaborate'), ' to advance research around single ventricle heart disease.'])],
```
Comment (cahrens) [inline src/components/common.js]: @kyuksel can you see about changing `heavyWrapper` so that it uses the `strong` tag instead of the `b` tag? With the HTML as-is, the screen reader experience is very poor. It stops reading before the first bold word, then you have to individually select each bit of text afterwards (and it doesn't state that it is emphasized). I haven't tried changing it to `strong`, but I'm hoping it will fix the issues.

https://help.siteimprove.com/support/solutions/articles/80000448460-accessibility-bold-vs-strong-and-italic-vs-emphasis

Comment (cahrens) [reply src/components/common.js]: Also, it feels odd to put the last part of the sentence in a different div. If the screen reader can read the whole thing seamlessly I guess it is OK, but semantically it feels wrong.

File: src/components/common.js (hunk 1 above)
Comment (nawatts) [reply src/components/common.js:56]: What was the motivation for re-formatting this?

File: src/components/common.js (hunk 3 above)
Comment (kyuksel) [reply src/components/common.js]: @cahrens Thanks for the catch! Addressed in [e1ce4a7](https://github.com/DataBiosphere/terra-ui/pull/3134/commits/e1ce4a7f3e3a57205a552839e3761913e205c83e).

Comment (nawatts) [reply src/components/common.js]: Agreed that `div` seems like the wrong element for wrapping a bit of text. I think `span` would be preferable. Does this need a DOM element at all?

File: src/components/common.js [hunk 4]
```diff
@@ -508,7 +508,12 @@ export const Switch = forwardRefWithName('Switch', ({ onChange, onLabel = 'True'
 })
 
 export const HeroWrapper = ({ showMenu = true, bigSubhead = false, children }) => {
-  const heavyWrapper = text => bigSubhead ? b({ style: { whiteSpace: 'nowrap' } }, [text]) : text
+  const heavyWrapper = text => bigSubhead ? strong({ style: { whiteSpace: 'nowrap' } }, [text]) : text
```
Comment (kyuksel) [inline src/components/common.js:511]: For improving accessibility as [described by @cahrens](https://github.com/DataBiosphere/terra-ui/pull/3134#discussion_r895971066)

File: src/components/common.js (hunk 3 above)
Comment (nawatts) [reply src/components/common.js]: Also, does the `collaborate` styling need to be repeated in both branches of the condition? Could only the `to advance research around single ventricle heart disease` phrase be conditionally appended?

Comment (kyuksel) [reply src/components/common.js]: > Agreed that div seems like the wrong element for wrapping a bit of text. I think span would be preferable. Does this need a DOM element at all?

No. Addressed in [fede1ef](https://github.com/DataBiosphere/terra-ui/pull/3134/commits/fede1efd03d410169d0c7c0d61d26ffaf56680fd).

Comment (kyuksel) [reply src/components/common.js]: > does the collaborate styling need to be repeated in both branches of the condition? Could only the to advance research around single ventricle heart disease phrase be conditionally appended?

Yes. Addressed in [894efab](https://github.com/DataBiosphere/terra-ui/pull/3134/commits/894efab69c93a594b2227af43333fdf65e06c2c6).

File: src/components/common.js [hunk 5]
```diff
@@ -528,10 +528,12 @@ export const HeroWrapper = ({ showMenu = true, bigSubhead = false, children }) =
         `${getAppName(true)} is a ${Utils.cond(
           [isTerra(), () => 'cloud-native platform'],
           [isFirecloud(), () => 'NCI Cloud Resource project powered by Terra'],
+          [isProjectSingular(), () => 'project funded by Additional Ventures and powered by Terra'],
           () => 'project powered by Terra'
         )} for biomedical researchers to `,
         heavyWrapper('access data'), ', ', heavyWrapper('run analysis tools'), ', ',
-        span({ style: { whiteSpace: 'nowrap' } }, ['and', heavyWrapper(' collaborate'), '.'])
+        h(Fragment, [span({ style: { whiteSpace: 'nowrap' } }, ['and', strong([' collaborate'])]),
+          isProjectSingular() && ' to advance research around single ventricle heart disease', '.'])
```
Comment (nawatts) [inline src/components/common.js]:
```suggestion
        span({ style: { whiteSpace: 'nowrap' } }, ['and', strong([' collaborate'])]),
        isProjectSingular() && ' to advance research around single ventricle heart disease', '.'
```

Now that this is inlined, I don't think the Fragment is necessary.

Comment (kyuksel) [reply src/components/common.js]: Thank you! Addressed in [d3bf3a7](https://github.com/DataBiosphere/terra-ui/pull/3134/commits/d3bf3a7fe23dc8c5c801f2a394f4424a52764570).

**Review by nawatts — APPROVED** [review]

File: src/components/common.js (hunk 1 above)
Comment (kyuksel) [reply src/components/common.js:56]: I have IntelliJ do the reformatting when I'm unsure about how to format code that I'm changing. I don't have a way to selectively apply that reformatting on a piece of a file. I hate to increase the PR diff but not reformatting now means I'd have to manually unapply reformatting every time I need to run the reformatter.

File: src/components/common.js [hunk 6]
```diff
@@ -528,10 +528,13 @@ export const HeroWrapper = ({ showMenu = true, bigSubhead = false, children }) =
         `${getAppName(true)} is a ${Utils.cond(
           [isTerra(), () => 'cloud-native platform'],
           [isFirecloud(), () => 'NCI Cloud Resource project powered by Terra'],
+          [isProjectSingular(), () => 'project funded by Additional Ventures and powered by Terra'],
           () => 'project powered by Terra'
         )} for biomedical researchers to `,
         heavyWrapper('access data'), ', ', heavyWrapper('run analysis tools'), ', ',
-        span({ style: { whiteSpace: 'nowrap' } }, ['and', heavyWrapper(' collaborate'), '.'])
+        span({ style: { whiteSpace: 'nowrap' } }, ['and', strong([' collaborate'])]),
```
Comment (cahrens) [inline src/components/common.js:535]: FYI, when I use VoiceOver it is breaking the text here at this span. It would be nice if it wasn't necessary to force `nowrap` around "and **collaborate**". (But don't feel like you have to keep working on this)

Comment (nawatts) [reply src/components/common.js:535]: Using a non-breaking space character between them instead of wrapping in a span might work.

Comment (cahrens) [reply src/components/common.js:535]: I played around a little with this in the browser, and it seems like the screen reader is trying to be smart about where to break the text (if you make the region wide enough so it can all render on one line, it doesn't split the content when reading).

File: src/components/common.js (hunk 1 above)
Comment (cahrens) [reply src/components/common.js:56]: @kyuksel you can select just a portion of the text and choose "Reformat Code".

image

File: src/components/common.js (hunk 6 above)
Comment (kyuksel) [reply src/components/common.js:535]: > Using a non-breaking space character between them instead of wrapping in a span might work.

Good to know. Were you thinking 
```javascript
`and&nbsp;${strong([' collaborate'])}`
```
?

Comment (kyuksel) [reply src/components/common.js:535]: > if you make the region wide enough so it can all render on one line, it doesn't split the content when reading

Do you mean by increasing `width: 575` on [line #527](https://github.com/DataBiosphere/terra-ui/pull/3134/files#diff-06c322acffe4a65c324584f6184927c7d4fca7e49530365c92671c0ea1fd30e1R527)?

File: src/components/common.js (hunk 1 above)
Comment (kyuksel) [reply src/components/common.js:56]: Oh wow -- so many pearls, you two 🤩

File: src/components/common.js (hunk 6 above)
Comment (nawatts) [reply src/components/common.js:535]: > Were you thinking
>
> `` `and&nbsp;${strong([' collaborate'])}` ``

I think React will escape `&nbsp;`, so you may have to use the Unicode character: `\u00a0`.

```js
'and\u00a0', strong(['collaborate']),
```

Comment (cahrens) [reply src/components/common.js:535]: To be clear, I'm not suggesting you increase the width. I was just trying it out to understand the effect of the width on how the screen reader was behaving.
